components/Organizers.tsx

```diff
@@ -0,0 +1,62 @@
+import { FC } from 'react';
+
+import * as cityData from '../pages/api/data';
+
+export const Organizers: FC = () => {
+  const organizers = Object.values(cityData.data).flatMap(
+    city => city.organiser,
+  );
+
+  return (
+    <div className="content">
+      <p className="h1 mb-4">社区组织者</p>
+      <div className="d-flex flex-wrap">
+        {organizers.map(item => (
+          <div
+            className="list_item mb-5 text-center"
+            key={item?.name}
+            style={{ marginBottom: '50px', width: '20%' }}
+          >
```

Comment:
```suggestion
    <Container>
      <h1 className="mb-4">社区组织者</h1>
      <Row as="ul" className="list-unstyled g-5" xs={2} sm={6} >
        {organizers.map(item => (
          <Col key={item.name} className="text-center">
```
1. https://react-bootstrap.github.io/docs/layout/grid/
2. https://getbootstrap.com/docs/5.3/content/typography/#unstyled